site/pages/smart-contracts/modular-account-v2/session-keys/removing-session-keys.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Removing Session Keys
+description: Removing Session Keys from your Modular Account V2
+---
+
+:::tip
+Session Keys are currently an experimental feature in the SDK. We are actively working on simplifying the usage, please note that there could be breaking changes as we improve this feature.
+:::
+
+# Removing session keys
+
+Removing session keys is done with the `uninstallValidation` method.
+
+```ts twoslash
+import { createModularAccountV2Client } from "@account-kit/smart-contracts";
+import {
+  installValidationActions,
+  getDefaultSingleSignerValidationModuleAddress,
+  SingleSignerValidationModule,
+  modularAccountAbi,
+} from "@account-kit/smart-contracts/experimental";
+import { LocalAccountSigner } from "@aa-sdk/core";
+import { sepolia, alchemy } from "@account-kit/infra";
+import { generatePrivateKey } from "viem/accounts";
+
+const client = (
+  await createModularAccountV2Client({
+    chain: sepolia,
+    transport: alchemy({ apiKey: "your-api-key" }),
+    signer: LocalAccountSigner.privateKeyToAccountSigner(generatePrivateKey()),
+  })
+).extend(installValidationActions);
+
+let sessionKeyEntityId = 1;
+
+// Removing a basic session key
```

Comment:
may be worth noting that you should (correct me if I am wrong) uninstall all hooks installed on the session key as well when uninstalling the session key